src/js/checkbox.js

```diff
@@ -0,0 +1,21 @@
+export const updateLocal = (getList) => {
+  localStorage.clear();
+  localStorage.setItem('to-do', JSON.stringify(getList));
+};
+
+export function checkList(check, listChek) {
```

Comment:
- [ ] Completed **tasks are not updating correctly in the local storage**. When _task one_ is checked as completed, the _task two_ status is rather updated in the local storage. Kindly ensure that the appropriate task's completed status is updated in the local storage when that particular task is checked

**Your app (on taschange status):**
![image](https://user-images.githubusercontent.com/12745474/140208572-e7eb2c6c-0ec2-4294-ad5d-7496ec7e7808.png)
s):

Reply:
thank you I didn't realize. But now I solved

Reply:
- [ ] @AtillaTahak , I appreciate the effort you made to fix this issue. Unfortunately, the issue persists. I think you need to test your `checkList` function as I suspect the issue is originated from there. Kindly make changes and also make sure the app works as intended before submitting for another code review request.

![Screenshot from 2021-11-04 02-19-46](https://user-images.githubusercontent.com/64894927/140232769-58604111-2d0f-4e37-86c8-3dd234efcf93.png)